Error model (#4)
svalinn/model_development_workflow: README.md

# Model Development Workflow Examples

## Purpose

> __Tutorials using specific models to explain and demonstrate important parts of the model workflow. By going through the tutorials one should understand all of the basic uses/functionality of SpaceClaim and Trelis/Cubit in the DAGMC workflow.__

Reviewer: ```suggestion
__These tutorials use specific models to demonstrate important steps of the DAGMC model workflow. By going through the tutorials one should understand all of the basic uses/functionality of SpaceClaim and Trelis/Cubit in the DAGMC workflow.__
```
Making it a full sentence

> ## Current Sections and Associated Files

Reviewer: Is there a better more descriptive title for this?

  * Characteristics of a Canonical Geometry:

>      - [Description of the overall purpose of this project and of each example individually](CharacteristicsOfCanon.md)

Reviewer: ```suggestion
     - [Overview of each example canonical geometry](CharacteristicsOfCanon.md)
```
Assuming we want this for our users, they will not be viewing this as a "project" so this wording suggestion is aiming for language more appropriate outside users.

>   * Simple Example of 4 Equal Cubes:

Reviewer: ```suggestion
  * Example of 4 Simple Cubes:
```
This is me being picky with consistency of naming the three sections. All others start w/ "Example.."

Also, I am not sure that "equal" is an important descriptor or the point of this particular example. More importantly it is just a simple example.

     - [Full tutorial](SimpleBlockX4.md)

>      - [Blocks by themselves before any edits in Cubit/Trelis](Simple100BlockX4.SAT)

Reviewer: ```suggestion
     - [The geometry before edits in Cubit/Trelis](Simple100BlockX4.SAT)
```

>      - [The part after adding the large volumes to be used as the graveyard and void volumes](Simple100BlockX4WGraveyardVoid.sat)

Reviewer: ```suggestion
     - [The geometry after the addition of a graveyard and implicit complement](Simple100BlockX4WGraveyardVoid.sat)
```
Try to stay consistent with terminology here. We should use "graveyard" and "implicit complement" instead of void volume at all. An implicit complement is not necessarily void.

Author: A few wording questions: 

1. Is geometry a generally accepted term for people in the simulation community? It seems more natural to me to use model or part coming from solidworks/autodesk, but I will change it.

2. Should I call it an implicit complement even when it is not being defined by the implicit complement tool? I agree that void volume isn't the right wording, but maybe something closer to "complementary volume".

Reviewer: @gonuke or @bohmt - Either of you have any insight into best terminology for question 1? @ngranda suggested "model" which I think is probably the best choice too. My original suggestion was "geometry" but I think "model" is a more appropriate term.

Reviewer: WRT to question 2- what do you mean by "implicit complement tool"? I have a large over arching comment that your tutorial should not be explicitly creating the "implicit complement" as that seems like an uncommon thing to do in practice.

>      - [Blocks and larger volumes after imprinting and merging](Simple100BlockX4WGraveyardVoidMerged.sat)

Reviewer: presumably all these files are going to be present in a separate PR, correct? So this section of text should come in with that PR rather than this one because it is not relevant to the Error Blocks section.

Reviewer: I also realize that I made suggestions on the block of text I recommended be removed. But those comments could be applied to your other relevant branches.

  
  * Example of Unclean Model with Common Errors:
     - [Full tutorial](ErrorBlocks.md)

>      - [Blocks in solidworks assembly format](ErrorBlocks.SLDASM)

Reviewer: Should Solidworks be capitalized?

     - [Blocks in ACIS (.sat) format](ErrorBlocks.SAT)
     - [Blocks in SpaceClaim format](ErrorBlocks.scdoc)
     
  * Example of Complex Model to Show Possible Complications

>      - [Full tutorial](ComplexAssembly.md)

Reviewer: same comment as above- these are not present in this PR so this text should come in with the relevant PR
